debug/ui-debug.js

```diff
@@ -0,0 +1,190 @@
+    {
+        title: "password only",
+        data: [
+            createCredentialsTooltipItem({ id: 'def', password: '1234567', username: 'Password for fill.dev', origin: { url: 'fill.dev' } })
+        ],
+        type: 'credentials.password',
+    },
+    {
+        data: [
+            createCredentialsTooltipItem({ id: 'somethingelse', username: '', password: '$124rg345h55aq67', [AUTOGENERATED_KEY]: true }),
+        ],
+        type: 'credentials.password.new',
+    },
+    {
+        data: [
+            createCredentialsTooltipItem({
+                credentialsProvider: 'bitwarden',
+                id: 'provider_locked',
+                password: '1234567',
+                username: 'dax@example.com',
+            }),
+        ],
+        type: 'credentials.password',
+    },
+]
+
+const identities = [
+    {
+        data: [
+            new IdentityTooltipItem({
+                firstName: 'Shane',
+                lastName: 'Osbourne',
+                id: '',
+                title: 'Main',
+            }),
+        ],
+        type: 'identities.firstName',
+    },
+];
+
+const creditCards = [
+    {
+        data: [
+            new CreditCardTooltipItem({ id: 'a', displayNumber: '1234', cardName: 'monzo', title: 'Monzo' }),
+            new CreditCardTooltipItem({ id: 'b', displayNumber: '8891', cardName: 'monzo', title: 'Mastercard' }),
+        ],
+        type: 'creditCards.cardNumber',
+    },
+]
+
+const emails = [
+    {
+        data: [],
+        title: 'in-context email signup only',
+        type: 'identities.emailAddress',
+        options: {
+            isIncontextSignupAvailable: () => true,
+        },
+    },
+    {
+        title: 'in-context email signup + item',
+        data: [
+            new IdentityTooltipItem({
+                id: 'personalAddress',
+                emailAddress: 'dax@example.com',
+                title: 'Block email trackers',
+            }),
+        ],
+        type: 'identities.emailAddress',
+        options: {
+            isIncontextSignupAvailable: () => true,
+        },
+    },
+    {
+        title: 'personal/private email address',
+        data: [
+            new IdentityTooltipItem({
+                id: 'personalAddress',
+                emailAddress: 'dax@example.com',
+                title: 'Block email trackers',
+            }),
+            new IdentityTooltipItem({
+                id: 'privateAddress',
+                emailAddress: '34534r_23edx@example.com',
+                title: 'Block email trackers and hide address',
+            }),
+        ],
+        type: 'identities.emailAddress',
+    },
+];
+
+const main = document.querySelector('main');
+
+credentials.forEach((iden, index) => createTooltip(iden, index));
+identities.forEach((iden, index) => createTooltip(iden, index));
+creditCards.forEach((iden, index) => createTooltip(iden, index));
+emails.forEach((iden, index) => createTooltip(iden, index));
+
+function createTooltip(item, index) {
```

Comment:
we make this smarter later of course, this is just a "V1 that's already useful" idea, to help Matteo's upcoming UI work